storage/connect/tabmysql.cpp

```diff
@@ -683,15 +684,21 @@ bool TDBMYSQL::MakeCommand(PGLOBAL g)
 
 
     // Make a lower case copy of the originale query
-    qrystr = (char*)PlugSubAlloc(g, NULL, strlen(Qrystr) + 5);
-    strlwr(strcpy(qrystr, Qrystr));
+    size_t qrystr_sz = strlen(Qrystr) + 5;
+    qrystr = (char*)PlugSubAlloc(g, NULL, qrystr_sz);
+    snprintf(qrystr, qrystr_sz, "%s", Qrystr);
+    strlwr(qrystr);
```

Comment:
rather than copy a string and force it to lower case after the copy. Do this all in one step with an appropriately wrapped function.

Reply:
The problem here is - all memory safe function we have (`safe_strcpy()` `snprintf()`) return int value (number of bytes copied) We can only wrapped unsafe functions (`strcpy()` as before or `sprintf()`) but this makes no sense here - we are trying to do opposite...